changelog.d/20231219_110746_nicolas.henin.md

```diff
@@ -0,0 +1,4 @@
+### @marlowe.io/runtime-lifecycle
+
+- [**Breaking Change**] `createContract` renamed to `submitCreateContract` ([PR#142](https://github.com/input-output-hk/marlowe-ts-sdk/pull/142))
```

Comment:
I don’t believe submitCreateContract is better than createContract… the name implies that you are submitting a CreateContract something. I feel that createContract in the context of a higher level package is descriptive enough. The omision is also arbitrary, if you want, this could have been called createSignAndSubmitContract, which is tedious to write, too long, I don’t like it. But why did you choose to keep submit and not sign?

Reply:
I'm not sure where I stand on this one... TBH I think it reveals that the API makes little semantic sense more than anything.

Reply:
`submitCreateContractTx` is what it ends up doing....  It is arbitrary... the same as saying `createContract` ... none of this solution is right... one is a partial description and the other is vague ... 
The exact description is `buildSignAndSubmitCreateContractTx`... The argument of saying :  "it is tedious to write/too long, so let's do something confusing instead", I don't agree with it... You have co-pilot, auto-completion in so many flavours....
I prefer us to have the truth in front of us everyday till we find something than hiding it...

Reply:
What I propose is to : 
1. change `submitCreateContractTx` (an the applyInput version ) to `buildSignAndSubmitCreateContractTx`
2. Ask the community to find a better name than : `buildSignAndSubmit***Tx`